### ISSUE TITLE
Unable to start a cluster with a custom storage pool

### ISSUE THREAD

**Issue by zabdulre** [oracle-cne/ocne#177]
There may be an issue using user-specified storage pools for cluster start.

To reproduce this issue, you must

1. Verify that ocne cluster start works without any providers.libvirt.storagepool value in the cluster config or defaults.yaml.
2. Then create a custom directory pool using virsh commands and an xml like the following 

`<pool type="dir">
  <name>my-pool</name>
  <target>
          <path>/some/path/to/store/volumes</path>
  </target>
</pool>`

5. Create a cluster config that has the providers.libvirt.storagepool field set to the newly created pool
6. Run cluster start with the created cluster config

8. `sudo systemctl status libvirtd.service` should show an error like the following  

`Cannot access storage file '/some/path/to/store/volumes/ocne-control-plane-1.qcow2' : Permission denied`

The likely issue is that running cluster start with a custom image pool causes ocne to create volumes with incorrect permissions or ownership.

**Comment by dkrasins** [comment]
This is likely an SELinux issue.  If this is reproducible with selinux=enforcing, try setting it to permissive and try again.

**Comment by George-Aeillo** [comment]
I was running into issues on a custom pool with selinux=enforcing, when I set selinux=permissive, I was able to spin up the cluster.

**Comment by George-Aeillo** [comment]
When I initially got this to work a couple of days ago with setting selinux=permissive, I was still creating a storage pool under the directory of /var/lib/libvirt. When I tried creating a storage pool under a different directory, I began running into permissions issue. When I edited the directory itself, its parent directories, and the storage pool permissions to allow the qemu user access to this directory, creating a storage pool in a different location worked as expected. This also requires selinux to be still set to permissive.
